packages/react-storybook/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@fluentui/react-storybook",
+  "version": "9.0.0-alpha.0",
+  "private": true,
+  "description": "Storybook addons and utils for Fluent UI React",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": false,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start": "just-scripts dev:storybook",
+    "start-test": "echo \"This is DEPRECATED instead use 'test --watch'\" && just-scripts jest-watch",
```

Comment:
I guess you can just get rid of it for new packages

```suggestion
```

Reply:
this is gonna be handled in batch/automation scripts per package when new Dx is ready